torch/quantization/quantize.py

```diff
@@ -229,6 +229,8 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     torch.nn.Conv2d: nnq.Conv2d,
     QuantStub: nnq.Quantize,
     DeQuantStub: nnq.DeQuantize,
+    # Generated modules:
+    torch.quantization.Add: torch.quantization.QAdd,
```

Comment:
why are these in quantization namespace?

Reply:
Yes, please update to nn.Add and nnq.Add